lib/pbench/test/unit/server/test_indexer.py

```diff
@@ -52,3 +55,27 @@ def test_value_types():
             templ, {"str": "abc", "int": 123, "float": 45.6789012, "other": []}
         )
         assert res == "abc_123_45.678901_%other%_UID"
+
+
+def test_init_indexing(monkeypatch, server_config, make_logger):
+    called = [False]
+
+    class MockPbenchTemplates:
+        def update_templates(self, *args, **kwargs):
+            called[0] = True
+            assert args[0] == "fake-es-obj", f"args={args!r}, kwargs={kwargs!r}"
+            assert not kwargs
+
+    class MockIdxContext:
+        def __init__(self, *args, **kwargs):
+            self.args = args
+            self.kwargs = kwargs
+            self.templates = MockPbenchTemplates()
+            self.es = "fake-es-obj"
```

Comment:
The `args` and `kwargs` attributes are not (AFAICT) referenced...are you sure you want to store them?

Reply:
Fixed.